worker/operations.py

```diff
@@ -517,14 +517,21 @@ def model_card(*args, **kwargs):
     text_file = paper_document_json.get(
         "text", "There is no documentation for this model"
     ).encode()
+    text_file = (
+        paper_document_json
+        .get("text", "There is no documentation for this model")
+    )
+
+    # TODO: Remove when no character limit exists for MIT
+    text_file = text_file[:9000]
```

Comment:
good TODO as a reminder.... i'll open an issue on MIT's repo so we can track this